src/components/Navbar.js

```diff
@@ -31,17 +34,27 @@ const Navbar = ({ toggleDarkMode, mode }) => {
     <AppBar
       position="static"
       style={{
-        background: mode === 'dark' ? '#0E6CDE' : '#0E6CDE',
+        background: mode === 'dark' ? '#373737' : '#111',
         marginBottom: '2rem',
       }}
     >
       <Toolbar>
-        <img
-          src={Logo}
-          alt="Logo"
-          style={{ height: '2.5rem', marginRight: '0.625rem' }}
-        />
-        <Typography variant="h6" component="div" sx={{ flexGrow: 1 }}>
+        <Link component={Link} to="/home">
```

Comment:
`Link` does not have a property called `component`. [https://reactrouter.com/en/main/components/link#link](https://reactrouter.com/en/main/components/link#link)

Right now this is just adding an HTML attribute to the rendered `a` tag but other than that it's not doing anything. It can be removed.

![](https://github.com/Code-the-Dream-School/dd-prac-team4-front/assets/9094098/6cf9b501-6260-4057-902c-e4ac3f67d93e)

```suggestion
        <Link to="/home">
```

Reply:
Only MUI elements have a `component` property: [https://mui.com/material-ui/guides/composition/#component-prop](https://mui.com/material-ui/guides/composition/#component-prop)

It's usually used in Material-UI when you want to tell it to render a particular MUI component as a different html tag or component. For example in this file, on line 51-55, we are rendering a MUI component, `Typography` and telling React to render it as a `Link` MUI component (which makes it an `a` tag).

Reply:
I see.. what I put here didn´t make any sence🫣